src/assets/css/global/typography.css

```diff
@@ -66,4 +82,9 @@ th {
 }
 .font-black {
   font-variation-settings: 'wght' 900;
-}
\ No newline at end of file
+}
+
+.font-numeric {
+  font-feature-settings: 'tnum';
+  letter-spacing: -0.5px;
```

Comment:
These are the attributes Pon wanted to use for all numbers.

@garethfuller do you like the name `font-numeric`?

Reply:
Yeah lgtm